config/config.go

```diff
@@ -1652,11 +1652,16 @@ func (c *Config) buildOutput(name, source string, tbl *ast.Table) (*models.Outpu
 	if err != nil {
 		return nil, err
 	}
+
+	bufferStrategy := c.Agent.BufferStrategy
+	if bufferStrategy == "disk_write_through" {
+		bufferStrategy = "disk"
+	}
```

Comment:
Should we do it the other way around i.e. map `disk` to `disk_write_through` and internally use the latter? This would keep things clear and self-descriptive in the actual code. What do you think?